azure-cs-credential-rotation-one-set/Azure.CredentialRotation.OneSet.csproj

```diff
@@ -0,0 +1,19 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <OutputType>Exe</OutputType>
+    <TargetFramework>netcoreapp3.1</TargetFramework>
+    <Nullable>enable</Nullable>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="Pulumi.Azure" Version="3.*" />
```

Comment:
Why is this one needed?

Reply:
`FirewallRule` and `AccessPolicy`

Reply:
Anything wrong with the NextGen `FirewallRule`? https://www.pulumi.com/docs/reference/pkg/azure-nextgen/sql/firewallrule/

I know we have an issue tracking the access policy. But you could define the `WebApp` resource before the `KeyVault` resource and then define web app's settings as a separate resource. https://www.pulumi.com/docs/reference/pkg/azure-nextgen/web/webapplicationsettings/

Reply:
Sorry that I ask so many questions, but does `WebApplicationSettings` get mapped from [WebApps_UpdateApplicationSettings](https://github.com/Azure/azure-rest-api-specs/blob/master/specification/web/resource-manager/Microsoft.Web/stable/2020-10-01/WebApps.json#L1012)? If so, shouldn't the class be named `WebAppApplicationSettings`? It sounds a bit repetitive, but it loses consistency.

Reply:
Hmm, it does, and it should. Mind opening an issue and I'll take a look at what goes wrong?

Reply:
https://github.com/pulumi/pulumi-azure-nextgen/issues/282

Reply:
Done.